documentation/book/security.adoc

```diff
@@ -1,61 +1,75 @@
 == Security
 
-The Apache Kafka project supports data encryption by means of the SSL/TLS protocol.
+The Strimzi project supports data encryption of communication between the different Kafka and Strimzi components (Kafka brokers, Zookeeper nodes, Kafka Connect, Strimzi Topic Operator) by means of the SSL/TLS protocol.
 This makes it possible to encrypt data transferred between brokers (interbroker communication) and between clients and brokers.
```

Comment:
This looks good. Just a few nits...

add an "and" before 'Strimzi Topic Operator'.